### ISSUE TITLE
Auto-Deployment

### ISSUE THREAD

**Issue by ddthj** [RLBot/core#29]
A longterm goal will be to have updates to rlbot auto-deploy.

**Comment by NicEastvillage** [comment]
Ideally, guilauncher will install `RLBotServer.exe` in `%localappdata%/RLBotGUI5` such that language interfaces will default to use this installation for launching matches if no other path to a `RLBotServer.exe` is given.

**Comment by VirxEC** [comment]
What do we think of putting `RLBotServer.exe` on `PATH`? This will make development exceptionally easy in the full release of v5, as you don't need to get another copy of of the exe. (Thinking of match starter scripts like we have in the python interface.)

Or, since the install path will be standardized, we can just point to the standard install location by default.

Just a thought.

**Comment by NicEastvillage** [comment]
> since the install path will be standardized, we can just point to the standard install location by default.

Yeah, that's what I said in my prev message. I think that is better than cluttering `PATH`.

**Comment by NicEastvillage** [comment]
Suggested file structure
```
%localappdata%/
  RLBot5/
    bin/
      RLBotServer.exe
      RLBotGUI.exe
    bots/
      botpack/
        ...
      local/
        ...   # bots created through the gui
```
